src/libecalc/core/graph_result.py

```diff
@@ -229,6 +226,9 @@ def operational_setting_used_id(timestep: datetime, operational_settings_used: T
         return operational_setting_id - 1
 
     @staticmethod
+    @Feature.experimental(
+        feature_description="Reporting requested pressures from compressor systems is an experimental feature."
```

Comment:
Er tanken her at vi ikke trenger å nevne compressor system? At `Reporting requested pressures is an experimental feature.`?

Reply:
Ja, det tror jeg ja - glemte den. Min tanke var opprinnelig at vi bare varslet ved bruk av system. Det er nå endret, bra spotta. Skal fikse.